features/mobile/people_aspects.feature

```diff
@@ -0,0 +1,43 @@
+@javascript
+Feature: adding and removing people from aspects
+    In order to add people to my contacts
+    As a mobile user
+    I want to add and remove people from my contacts
+
+    Background:
+      Given following users exist:
+        | username   |
+        | bob        |
+        | alice      |
+      And I toggle the mobile view
+      And I am on the home page
+      And I sign in as "bob@bob.bob" on the mobile website
+
+    Scenario: verify different states of the cover button
+      When I visit the user profile "alice"
```

Comment:
https://github.com/diaspora/diaspora/blob/develop/features/desktop/connects_users.feature#L11 No need for a new step